go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/containerd/containerd v1.6.6 // indirect
 	github.com/cyphar/filepath-securejoin v0.2.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/deckarep/golang-set/v2 v2.1.0 // indirect
```

Comment:
I'm curious why this says //indirect as we are directly referencing in the code?

Reply:
I guess it is because we say v2, not v2.1.0. Actually I prefer to set the exact version, I didn't find a way to do it.